src/algebraic_topology/simplicial_object.lean

```diff
@@ -26,6 +31,15 @@ This is the category of contravariant functors from `simplex_category` to `C`. -
 def simplicial_object := simplex_categoryᵒᵖ ⥤ C
 
 namespace simplicial_object
+
+instance [has_limits C] : has_limits (simplicial_object C) :=
```

Comment:
Hmm, should this be done shape for shape, because maybe `C` doesn't have all limits, but still has certain interesting ones?

Reply:
Yeah good point. I'll change this soon.

Reply:
Done!